test/src/runner/testRunTestsuite.js

```diff
@@ -148,10 +149,10 @@ describe('testRunTestSuite', function () {
 
     let globals = {
       reporter(results, cb) {
-        assert.ok('test/sample' in results.modules);
-        assert.ok('mixed/sample' in results.modules);
-        assert.ok('demoTest' in results.modules['test/sample'].completed);
-        assert.ok('demoTestMixed' in results.modules['mixed/sample'].completed);
+        assert.ok(`test${path.sep}sample` in results.modules);
```

Comment:
can we create a test util function for this?

Reply:
for `path.sep` ?

Reply:
yes, we can have a function, which will replace `test/sample` with `test${path.sep}sample`

Reply:
I don't think that is necessary for a specific string. This is more generic.